contrib/masterfiles-stage/README.org

```diff
@@ -60,7 +60,7 @@ earlier versions of git may or may not work.
 :ID:       2aeaaa9b-1229-4c14-b130-6d86e370de42
 :END:
 Installation simply consists of saving =masterfiles-stage.sh= and its supporting
-=common.sh= to the same directory on your policy server, making sure that
+=common.sh= and =ssh-wrapper.sh= to the same directory on your policy server, making sure that
```

Comment:
Did you pull ssh-wrapper.sh from nova or mission-portal?

Reply:
there is only one in nova AFAICT

https://github.com/cfengine/nova/tree/master/api/http/dc-scripts

Can't find one in mission-portal.

Reply:
may be more files we need to pull/sync. Will re-test using these current docs and adjust. :+1:

Reply:
yes, in order to satisfy the examples we also need `git-askpass.sh`